README.md

```diff
@@ -33,13 +33,7 @@ module.exports = {
 "@netlify/eslint-config-node/.prettierrc.json"
 ```
 
-- Add the following `commitlint.config.js` to the root of the project:
-
-```js
-module.exports = { extends: ['@commitlint/config-conventional'] }
-```
-
-- Copy the `.editorconfig` and `.gitattributes` files relativity to the root of the project.
+- Copy the `commitlint.config.cjs`, `.editorconfig` and `.gitattributes` files relativity to the root of the project.
```

Comment:
Saw it was copy paste but should `s/relativity/relative/`? 🤔 
```suggestion
- Copy the `commitlint.config.cjs`, `.editorconfig` and `.gitattributes` files relative to the root of the project.
```

Reply:
Ah, it's been automerged :-/

Good catch, I have opened a PR at https://github.com/netlify/eslint-config-node/pull/438 to fix it.